README.md

```diff
@@ -306,6 +306,11 @@ These open source projects will help you to build a bot in order to automate you
 
 * [Hubot](https://hubot.github.com/) [adapter](https://github.com/athieriot/hubot-yammer) (CoffeeScript, Node.js)
 
+### Zulip
+
+* [Errbot](http://errbot.io/) [adapter](https://github.com/zulip/errbot-backend-zulip) (Python)
```

Comment:
@jayvdb wow, awesome!

Could you please add `+` (forgot to add it for `Yammer`)? 🙏
So, it'll look like:

```
* Errbot + adapter (Python)
* Hubot + adapter (CoffeeScript, Node.js)
```

Reply:
:/  https://github.com/exAspArk/awesome-chatops/blob/master/CONTRIBUTING.md#for-bots-section says not to add the `+` , so I guess that needs fixing?

Reply:
Yep, right 💯 
I'll update it